src/components/Contentful/DynamicRichText.vue

```diff
@@ -0,0 +1,41 @@
+<script>
+import KvContentfulImg from '~/@kiva/kv-components/vue/KvContentfulImg';
+
+/**
+* This component takes an html string and renders it with vue
+* */
+
+export default {
+	props: {
+		html: {
+			type: String,
+			default: ''
+		}
+	},
+	computed: {
+		dynamicComponent() {
+			return {
+				template: `<div>${this.html}</div>`,
+				components: {
+					KvContentfulImg
+				},
+				props: [
+					/* define props as needed */
+				]
+			};
+		}
+	},
+	render(createElement) {
+		return createElement(
+			'div', /* Wrap output in a div since Vue templates must have 1 root element */
```

Comment:
This wrapper this is for this `DynamicRichText.vue` component, which unfortunately is extraneous, but I couldn't figure out how to get this component to work without it